src/composables/Checklist.ts

```diff
@@ -0,0 +1,730 @@
+type ChecklistItem = {
+  name: string
+  source?: string
+}
+
+type StorageItem = ChecklistItem & {
+  bagSlots: number
+}
+
+type CapacityPouchesItem = ChecklistItem & {
+  cycle: string
+}
```

Comment:
Love these types 💯

Reply:
Should I add them again? I removed them when I found the existing `Item` and `ItemGroup` types 😅

Reply:
Oh man it's been a while - yeah let's use the existing types. Can rename them if it makes it more clear.